Backpack/BottomSheet/Classes/BottomSheet.swift

```diff
@@ -225,26 +229,20 @@ public final class BPKBottomSheet: NSObject {
 }
 
 extension BPKBottomSheet: FloatingPanelControllerDelegate {
-    final class IntrinsicLayout: FloatingPanelIntrinsicLayout {
-        func backdropAlphaFor(position: FloatingPanelPosition) -> CGFloat {
-            return Constants.backdropAlpha
-        }
-    }
-    
     public func floatingPanel(
         _ viewController: FloatingPanelController,
         layoutFor newCollection: UITraitCollection
-    ) -> FloatingPanelLayout? {
+    ) -> FloatingPanelLayout {
```

Comment:
this was nillable without the need for it?? yeez.

It's a breaking change tho, and we need to treat this as a major release

Reply:
The library has changed the protocol so the layout is no longer optional. 
This should not be consumed by consumers I assume. 

In any case we probably want to go for a major update, because we jumped to v2 of floating panel